gobblin-data-management/src/test/java/org/apache/gobblin/data/management/copy/writer/FileAwareInputStreamDataWriterTest.java

```diff
@@ -402,8 +423,29 @@ public void testCommit() throws IOException {
     // previously existing paths should not have permissions changed
     fileStatus = this.fs.getFileStatus(existingOutputPath);
     Assert.assertEquals(fileStatus.getPermission(), existingPathPermission);
-
+    verifyAclEntries(writer, pathToAclEntries, expectedOutputPath);
```

Comment:
reqs-wise, how does this feature treat ACLs on parent dirs that it's creating--will they be preserved or not?  a unit test should codify "the spec".  hence, either way, shouldn't we exercise that scenario to assert it's handled in the proper way?